documentation/slate/source/includes/realtime.md

```diff
@@ -0,0 +1,206 @@
+<a name="realtime"></a> Real time integration in Navita
+===============================================
+
+Several endpoints can integrate real time information in their responses. In the response received, a disruption will be present and additional information will be provided if the parameter `data_freshness` is set to `realtime`.
+
+<aside class="notice">
+Real time isn't available on every coverages for Navitia. For real time to be available for a client, it needs to provide real time info about its network to Navitia, which can be provided through several means (GTFS-RT for ex or proprietary format like the SNCF client).
+</aside>
+
+The effect of a disruption can be one of the following:
+<ul>
+	<li>SIGNIFICANT_DELAYS</li>
+	<li>REDUCED_SERVICE</li>
+</ul>
+
+For each one of these effects, here's how the Navitia responses will be affected over the different endpoints. A link to the API
+
+
+## Trip delayed
+
+The effect of the disruption is `SIGNIFICANT_DELAYS`. It means that the train will arrive late at one or more stations in its journey.
+
+```
+{
+    "amended_arrival_time": "093200",
+    "amended_departure_time": "093300",
+    "arrival_status": "delayed"
+    "base_arrival_time": "092200",
+    "base_departure_time": "092300",
+    "cause": "Panne d'un aiguillage",
+    "departure_status": "delayed",
+    "is_detour": false,
+    "stop_point": ⊕{7 items},
+    "stop_time_effect": "delayed",
+}
+```
+
+In the disruption, the delay can be found in the list of "impacted_stops" with the departure/arrival status set to "delayed". See the [disruption](#disruption) objects section for its full content and description.
+
+### Journeys
```

Comment:
CSS with front-end dev?

Reply:
The power of <div></div> :smiling_imp:

Reply:
No, to have the description and the shell of one endpoint facing each other. And it does that!